pravega/pravega-sensor-collector
# Updating READMe file

Change log description
Update README to include new field for paramaters table. The new field tells user whether the parameter is optional or mandatory for running psc service

Purpose of the change
Update optional / mandatory fields.

What the code does?
Adds new column in readme to denote mandatory/optional parameter.

How to verify it
Update parameter file (PravegaSensorCollector.xml) as per the parameters table and psc should run without error.
